### ISSUE TITLE
Focus next input with custom button

### ISSUE THREAD

**Issue by ravimaurya-nickelfox** [bamlab/react-native-formik#118]
I have UI that has iOS keyboard navigation options to focus next/previous input. I am unable to focus the inputs using the button.
![image](https://user-images.githubusercontent.com/50916102/60007274-9d983b00-968f-11e9-88fd-f888fe0b6add.png)

### Can anyone help me with any method available to focus the input?

**Comment by thienlhh** [comment]
I have a same question

**Comment by atipezda** [comment]
Up
